src/config.rs

```diff
@@ -13,6 +13,8 @@ pub const STALLED_TIMEOUT: usize = 10;
 pub const MAX_MESSAGES_PER_UNIT: usize = 128;
 pub const MAX_PARENT_PER_UNIT: usize = 16;
 
+pub const STORAGE: &str = "sqlite";
```

Comment:
we only support sqlite, so there is no need to declare this const. and update all the implementations to sqlite specific should be fine